src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -222,23 +228,18 @@ export class BitcoinAdapter extends BaseProtocolAdapter {
           value, // Amount
           transactionInHex: await rawTransactionBody.text(),
         };
-      });
+      }));
 
-    const utxoDetails = await Promise.all(fetchHexPromises);
-
-    const amountInSatoshi = toSatoshi(amountInBtc);
-    const feeInSatoshi = toSatoshi(options.fee);
-    let totalBalance: number = 0;
-    let hasSufficientBalance = false;
+    // Sort UTXOs by ascending absolute difference from the amount to send
+    const spendableUtxos = fullUtxos.sort(
+      (a, b) => Math.abs(a.value - (amountInSatoshi + feeInSatoshi))
+              - Math.abs(b.value - amountInSatoshi + feeInSatoshi),
```

Comment:
You could remove parentheses or add them in the second line to have same style or you could extract the sum of amount and fee to common variable and then subtract it
```suggestion
      (a, b) => Math.abs(a.value - (amountInSatoshi + feeInSatoshi))
              - Math.abs(b.value - amountInSatoshi + feeInSatoshi),
```

Reply:
Fixed as per Bart's suggestion.